static/src/js/actions/exportSearch.js

```diff
@@ -56,6 +57,35 @@ export const exportSearch = (format) => (dispatch, getState) => {
       }
     }`
 
+  const searchExportParams = {
+    itempath: 'collections.items',
```

Comment:
keys should be camelCase

Reply:
fixed with https://github.com/nasa/earthdata-search/pull/1630/commits/039f6b6d917e7671171884f58c3edd0dad60f7f3